docs/apm/correlations.asciidoc

```diff
@@ -94,7 +99,7 @@ closer to a diagnosis.
 
 [discrete]
 [[correlations-customize-fields]]
-====  Customize fields
+=====  Customize fields
 
 Correlations are only as good as the data they're searching for.
 By default, a handful of attributes commonly known to cause performance issues are included.
```

Comment:
I realize this is on me, but the line right below this says "to need to" instead of "need to"

Reply:
Done!